hal/include/HAL/I2C.h

```diff
@@ -8,8 +8,9 @@
 #pragma once
 
 #include <stdint.h>
+#include "HAL/Types.h"
```

Comment:
Since this is now including types, will you remove the <stdint.h> include, since that is in types.